### PR TITLE
adds postgres instructions

### PR DESCRIPTION
**Pull request by edublancas**
added two new sections showing how to get the postgres connector and how to connect to a running postgres instance

### CODE REVIEW

**Review by yafimvo — CHANGES_REQUESTED** [review]
Not sure if this is the right place for it or should we need to open a new issue.

It generates a horizontal scroller since the generated table is not contained in the page

image

**Comment by edublancas** [comment]
> Not sure if this is the right place for it or should we need to open a new issue.
> 
> It generates a horizontal scroller since the generated table is not contained in the page

we're using a template, so the best place is here: https://github.com/executablebooks/sphinx-book-theme

File: doc/howto/postgres-connect.ipynb [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "b2745e92",
+   "metadata": {},
+   "source": [
+    "```{tip}\n",
+    "If you have issues, check out our [installation guide](postgres-install) or [message us on Slack.](https://ploomber.io/community)\n",
+    "```\n",
+    "\n",
+    "You also need Docker installed and running do start the PostgreSQL instance."
```
[the remark this answers is not in the document]
Comment (edublancas) [reply doc/howto/postgres-connect.ipynb:76]: fixed

File: doc/howto/postgres-install.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+To connect to a PostgreSQL database from Python, you need a client library. We recommend using `psycopg2`, but there are others like `pg8000`, and `asyncpg`. JupySQL supports the [following connectors.](https://docs.sqlalchemy.org/en/14/dialects/postgresql.html#dialect-postgresql)
+
++++
+
+### Installing `psycopg2`
+
+The simplest way to install `psycopg2` is with the following command:
+
+```sh
+pip install psycopg2-binary
```
[the remark this answers is not in the document]
Comment (edublancas) [reply doc/howto/postgres-install.md:12]: the problem is that many people don't use conda (only pip), so I wanted to include the pip command. We're suggesting the `-binary` package, which shouldn't lead to issues. do you think we should also remove this reference?

> stall `psycopg2` is with the 

so they don't attempt to do "pip install psycopg2"?

[the remark this answers is not in the document]
Comment (yafimvo) [reply doc/howto/postgres-install.md:12]: I think let's keep it and see if people are struggling with this. we give 2 alternatives so I think it's fair.

**Comment by yafimvo** [comment]
> we're using a template, so the best place is here: https://github.com/executablebooks/sphinx-book-theme

I'll open an issue there

File: doc/howto/postgres-install.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (edublancas) [reply doc/howto/postgres-install.md:12]: sure. are we good to merge then?

**Review by yafimvo — APPROVED** [review]

**Pull request merged**